cta_names_mappings.json

```diff
@@ -453,6 +454,7 @@
   "TEMP.Hermit": {
   },
   "Kimsuky": {
+    "subgroups": ["TA406", "TA408", "TA427"]
```

Comment:
Given you added a country to Kimusky in the md file, does it also need that country field here in the JSON?

Reply:
It does, I've now corrected that.